src/pages/en/guides/styling.md

```diff
@@ -114,13 +101,57 @@ You can achieve the same by using the `:global()` function at the root of a sele
 </style>
 ```
 
-It’s recommended to only use this in scenarios where a `<link>` tag won’t work. It’s harder to track down errant global styles when they’re scattered around and not in a central CSS file.
+> ⚠️ These global styles apply throughout your entire project! 
+
+It may be easy to lose track of which Astro component is defining styles globally, and harder to troubleshoot errant global styles when they’re scattered around and not in a central CSS file. So, this method is recommended only when you cannot otherwise apply global styling via an import or `<link>`.
+
+
+## External Styles
+
+There are two ways to resolve global stylesheets: an ESM import for files located within your project source, and an absolute URL link for files in your /public directory, or hosted outside of your project.
 
-📚 Read our full guide on [Astro component syntax][astro-component] to learn more about using the `<style>` tag.
 
-### Variables in Scripts & Styles
+### Import a Global Stylesheet
 
-In Astro v0.21+, _serializable_ server-side variables can be passed into client-side `<style>` or `<script>`.
+Astro detects these CSS imports and then builds, optimizes, and adds the CSS to the page automatically. 
```

Comment:
I can see why you might lead with this line as it advertises some positive things, but logically I think it might make more sense just after the example. Otherwise maybe needs a bit of work as 

```
Import a Global Stylesheet

...these imports
```

isn’t clear about what “these” imports are.